yarn-project/circuits.js/src/barretenberg/crypto/pedersen/pedersen.ts

```diff
@@ -10,6 +10,7 @@ import { deserializeArrayFromVector, deserializeField, serializeBufferArrayToVec
  * @param lhs - The first hash.
  * @param rhs - The second hash.
  * @returns The new 32-byte hash.
+ * @deprecated Don't call pedersen directly in production code. Instead, create specific nicely-called WASM functions.
```

Comment:
```suggestion
 * @deprecated Don't call pedersen directly in production code. Instead, create suitably-named functions for specific purposes.
```

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/2724/commits/4b722a5d79143fa4138c7c7fded119945b23d5f7